clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -2619,6 +2619,9 @@ private ConfigEntry.ConfigSource configSource(DescribeConfigsResponse.ConfigSour
             case DYNAMIC_BROKER_LOGGER_CONFIG:
                 configSource = ConfigEntry.ConfigSource.DYNAMIC_BROKER_LOGGER_CONFIG;
                 break;
+            case DYNAMIC_CLIENT_METRICS_CONFIG:
```

Comment:
I think I would make this match the `TOPIC_CONFIG` so that this value is actually `CLIENT_METRICS_CONFIG`. They're all dynamic, just like topic configs.

Reply:
Make sense, done.